### PR TITLE
Bori Patch

### PR DESCRIPTION
**Pull request by Tostov**
## Additions

Adds patch stuff for the bori race mod.
Should cover everything currently.

## Testing

Check tests you have performed:
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Generally looks good, with a couple of small errors to resolve.

File: Patches/Bori Race/ThingDefs_Misc/Weapons.xml [hunk 1]
```diff
@@ -0,0 +1,914 @@
...
+				<xpath>Defs/ThingDef[defName="BD_Autopistol"]/weaponTags</xpath>
+				<value>
+					<li>CE_OneHandedWeapon</li>
+				</value>
+			</li>
+			
+			<!-- ========== Auto Heavy Rifle =========== -->
+			
+			<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+			<defName>Miho_Weapon_RifleDefence</defName>
```
[the remark this answers is not in the document]
Comment (Tostov) [reply Patches/Bori Race/ThingDefs_Misc/Weapons.xml]: Fixed.

File: Patches/Bori Race/ThingDefs_Misc/Bori_Explosives.xml [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Bori race</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+
+		<!-- ========== Remove Recipe Maker & Verbs ========== -->			
+		<li Class="PatchOperationRemove">
+			<xpath>Defs/ThingDef[defName="BD_GrenadeFrag"]//recipeMaker</xpath>
```
[the remark this answers is not in the document]
Comment (Tostov) [reply Patches/Bori Race/ThingDefs_Misc/Bori_Explosives.xml:13]: Not a typo. The double slash is used to remove the entire node. If not present you get the following (mostly harmless afaik) error on startup:
Config error in BD_GrenadeFrag: has a recipeMaker but no costList or costStuffCount.
UnityEngine.StackTraceUtility:ExtractStackTrace ()
(wrapper dynamic-method) Verse.Log:Verse.Log.Error_Patch1 (string)
Verse.DefDatabase`1<Verse.BuildableDef>:ErrorCheckAllDefs ()
System.Reflection.MonoMethod:Invoke (object,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo)
System.Reflection.MethodBase:Invoke (object,object[])

**Review by N7Huntsman — APPROVED** [review]
Looks good!

**Pull request merged**